projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -129,6 +129,7 @@ export function effectResubscriptionHandler&gt;T extends Action&lt;(
   );
 }
 
+- im not sure what should be done with that one
```

Comment:
I tink this page needs to undergo some more changes 😅
We could update this example to use `bootstrapApplication` or leave it as is for later.
(but remove this comment please). 

```ts
bootstrapApplication(
  AppComponent,
  {
    providers: [
      {
        provide: EFFECTS_ERROR_HANDLER,
        useValue: effectResubscriptionHandler,
      },
      {
        provide: ErrorHandler, 
        useClass: CustomErrorHandler 
      }
    ],
  }
)
```